docs/books/admin_guide/09-backups.md

```diff
@@ -166,188 +166,204 @@ There are two different storage methods:
 
 The `tar` command allows saving on several successive media (multi-volume options).
 
-It is possible to extract all or part of a backup.
+You can extract some or all of the files from the archive file.
 
-`tar` implicitly backs up in relative mode even if the path of the information to be backed up is mentioned in absolute mode. However, backups and restores in absolute mode are possible.
+When using `tar`, please note that it has two saving modes:
 
-### Restoration guidelines
+* **Relative mode** (default): Remove the leading character '/' from the file to be archived. Even if you have added the file to be archived with an absolute path, the leading character "/" will still be removed in this mode.
+* **Absolute mode**: Keep the leading character '/' and include it as part of the file name. To enable this save mode, you need to use the `-P` option. In this mode, all files you add must be represented as absolute paths.
 
-The right questions to ask are:
+When you use `tar`, you will encounter suffixes such as `.tar.gz`, `.tar.xz`, `.tar.bz2`, which indicate creating an archive first (categorizing related files as a single file), and then compressing the file using the relevant compression type or compression algorithm.
 
```

Comment:
As far is this goes, it's fine, but why are you deleting the section "The right questions to ask are:" and all of its subsequent content?